nilearn/tests/test_numpy_conversions.py

```diff
@@ -235,9 +235,12 @@ def test_as_ndarray():
 
 def test_csv_to_array():
     # Create a phony CSV file
-    with tempfile.NamedTemporaryFile(suffix='.csv', mode='wt') as csv_file:
-        csv_file.write('1.,2.,3.,4.,5.\n')
-        csv_file.flush()
-        assert_true(np.allclose(csv_to_array(csv_file.name),
```

Comment:
> I think that this failure you could have addressed with a 'del csv_file' at the end of the with statement.

IMHO, the problem here was coming from `csv_to_array(csv_file.name)` which was opening another file descriptor on `csv_file.name`